### ISSUE TITLE
Cannot use VoiceOver for author and time in text messages

### ISSUE THREAD

**Issue by bear101** [BearWare/TeamTalk5#248]

**Comment by OzancanKaratas** [comment]
Also event log is being seen on the private text message window. This problem should be resolved.
